cmd/juju/cloud/updatepublicclouds.go

```diff
@@ -83,6 +84,17 @@ func (c *updatePublicCloudsCommand) Info() *cmd.Info {
 	})
 }
 
+type PublicCloudsAccessDetails struct {
+	publicSigningKey string
+	publicCloudURL   string
+}
+
+// PublicCloudsAccess contains information about
+// where to find published public clouds details.
+func PublicCloudsAccess() PublicCloudsAccessDetails {
+	return PublicCloudsAccessDetails{keys.JujuPublicKey, "https://streams.canonical.com/juju/public-clouds.syaml"}
```

Comment:
nit: we should use named fields.
```suggestion
	return PublicCloudsAccessDetails{
		publicSigningKey: keys.JujuPublicKey, 
		publicCloudURL: "https://streams.canonical.com/juju/public-clouds.syaml",
	}
```

Reply:
I am happy to update it but this code styling is a bit heavy. For example, GoLand implies the keys anyway.

Reply:
I believe we should, there are downsides to not actually knowing what fields we're using whilst reading the code.

It's one of our aims to add these at some point.